app/models/ems_event.rb

```diff
@@ -146,6 +146,34 @@ def self.process_vm_in_event!(event, options = {})
         event[:vm_name] ||= vm.name
       end
     end
+
+    event_type = event[:event_type]
+    # both event types are Rhev specific
+    if (event_type == 'USER_ADD_VM' || event_type == 'USER_ADD_VM_FINISHED_SUCCESS') && event[:vm_or_template_id].nil?
```

Comment:
>> In vmware you would do a targeted refresh on the Host that that new vm is on

>Oh no actually it will do a targeted refresh on the new VM

So I was mistaken about this ^^, on a new vm event we do switch the target to the host the VM is on